badger/cmd/read_bench.go

```diff
@@ -64,18 +65,15 @@ func init() {
 	readBenchCmd.Flags().StringVarP(
 		&duration, "duration", "d", "1m", "How long to run the benchmark.")
 	readBenchCmd.Flags().IntVar(
-		&sampleSize, "sample-size", 1000000, "Keys sample size to be used for random lookup.")
+		&ro.sampleSize, "sample-size", 1000000, "Keys sample size to be used for random lookup.")
 	readBenchCmd.Flags().BoolVar(
-		&keysOnly, "keys-only", false, "If false, values will also be read.")
+		&ro.keysOnly, "keys-only", false, "If false, values will also be read.")
 	readBenchCmd.Flags().BoolVar(
-		&readOnly, "read-only", true, "If true, DB will be opened in read only mode.")
-	readBenchCmd.Flags().StringVar(
-		&loadingMode, "loading-mode", "mmap", "Mode for accessing SSTables and value log files. "+
-			"Valid loading modes are fileio and mmap.")
+		&ro.readOnly, "read-only", true, "If true, DB will be opened in read only mode.")
 	readBenchCmd.Flags().BoolVar(
-		&fullScan, "full-scan", false, "If true, full db will be scanned using iterators.")
-	readBenchCmd.Flags().Int64Var(&blockCacheSize, "block-cache", 0, "Max size of block cache in MB")
-	readBenchCmd.Flags().Int64Var(&indexCacheSize, "index-cache", 0, "Max size of index cache in MB")
+		&ro.fullScan, "full-scan", false, "If true, full db will be scanned using iterators.")
+	readBenchCmd.Flags().Int64Var(&ro.blockCacheSize, "block-cache", 1024, "Max size of block cache in MB")
```

Comment:
Shouldn't this be set to default value `256MB`?

Reply:
Sure. Fixed it.

Reply:
There is one more in the write bench. I missed publishing that.

Reply:
fixed